electron/app/components/ImageContainerHeader.tsx

```diff
@@ -37,12 +37,13 @@ const Wrapper = styled.div`
   }
 `;
 
-const ImageContainerHeader = ({
-  datasetName,
-  total = 0,
-  showSidebar,
-  onShowSidebar,
-}: Props) => {
+const ImageContainerHeader = ({ showSidebar, onShowSidebar }: Props) => {
+  const totalCount = useRecoilValue(selectors.totalCount);
+  const filteredCount = useRecoilValue(selectors.filteredCount);
+  const countStr =
+    filteredCount !== null
```

Comment:
Storybook error: `Cannot read property 'toLocaleString' of undefined`
It may be safest to change the selectors to always return a number; otherwise, checking `typeof filterCount === 'number'` might be better (since that's really what determines whether toLocaleCount will work)

Reply:
On it